.prow.yaml

```diff
@@ -310,7 +310,7 @@ postsubmits:
             - "-c"
             - |
               executor --context=${PWD}/images/releaser \
-                --dockerfile=Dockerfile --destination=nephio/releaser:${PULL_BASE_REF} \
+                --dockerfile=Dockerfile --destination=nephio/releaser:${BUILD_ID} \
```

Comment:
Where is ${BUILD_ID} defined? Thanks

Reply:
Those are Prow job env variables: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables

Reply:
Thanks